Tensile/Source/client/include/DataInitialization.hpp

```diff
@@ -57,6 +57,16 @@ namespace Tensile
             Count
         };
 
+        enum class BoundCheckMode
```

Comment:
For consistency, this should be `BoundsCheckMode`

Reply:
@sdquiring done